### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by WooHaem** [vercel/community#3807] [Account Issue] [answered]
### Question

I tried to sign up, but my account was blocked for unknown reasons. This is my first attempt at membership registration and I do not know English. What kind of mechanism is this? Can someone give me an easy explanation? I haven't done anything to vercel. I am very curious why my account was blocked. I have sent an inquiry email and am waiting for a reply, but as you can see in the article below, others are still experiencing the same symptoms. This user experience is very inconvenient.

In addition, I have tried all the methods I have seen in other posts, but I have not found any change. My account was still blocked, and I'm very curious as to why. I hope to receive a quick response as this is a problem that not only me but others are experiencing.
![err](https://github.com/vercel/community/assets/87007506/52901bca-9953-45fd-9bbf-a8f5c39d865e)

**Comment by amyegan** [comment, marked as answer]
Sorry you ran into a problem with sign up, @WooHaem. Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy.
